app/background/wallet/service.js

```diff
@@ -412,14 +412,15 @@ class WalletService {
   send = (to, amount, fee) => this._ledgerProxy(
     () => this._executeRPC('createsendtoaddress', [to, Number(amount), '', '', false, 'default']),
     async () => {
-      await this.client.send(this.name, {
+      const res = await this.client.send(this.name, {
```

Comment:
This response was removed in #252 .

To prevent future regression, let's start to add unit tests around critical code path #273